uui/components/pickers/PickerToggler.tsx

```diff
@@ -68,6 +72,7 @@ function PickerTogglerComponent<TItem extends string, TId>(props: PickerTogglerP
     return (
         <UuiPickerToggler
             { ...props }
+            id={ props?.id }
```

Comment:
Is it this props will no be spreaded by { ...props }?

Reply:
fixed

Reply:
still doesn't fixed

Reply:
sorry, already fixed